Feature/back 6 anynote django application

egorgur opened the pull request:
Anynote main django application layout

```sh
src
    ├───apps
    │   ├───anynote
    │        ├───api
    │        ├───migrations
    │   ├───authentication
    │        ├───api
    │        ├───migrations
    ├───config
    │   ├───settings
    │        ├───templates
    ├───general
    │   ├───tests
    │        ├───fixtures
    │   ├───utils
    ├───tests
```

egorgur committed:
Init anynote django app and create anynote api directory

egorgur committed:
Establish API's urls routing patterns

egorgur committed:
Add src/api/urls urlpatterns to src/config/urls

egorgur commented on docker-compose.yaml:
Ignore this, there are actual changes

IWorkForFood commented on src/config/settings/logging.py:
Is not it less readable

```
"handlers": {
        "console": {"level": "INFO", "class": "rich.logging.RichHandler", "formatter": "standard", "filters": []},
```
than this one

```
"handlers": {
        "console": {"level": "INFO",
         "class": "rich.logging.RichHandler", 
         "formatter": "standard", "filters": []
         },
```

The same question applies to this part:

```
"formatters": {"standard": {"format": "%(asctime)s %(levelname)s %(name)s %(message)s"}},
```
and to this part:

```
"filters": {"info_filter": {"()": "src.config.settings.InfoFilter"}},
```

egorgur replied:
I agree with you and i tried to make it unfold manually. But the ruff pre-commit formatter makes it fold back. Also just standart ctrl + alt + l also makes them shrink.
So i could not unfold those dictionaries without automatically folding them back by the linter

egorgur replied:
resolved in IRL

egorgur committed:
Restucturize the django applications into an apps/ module

Now the src/ module contains the apps/ module.
The apps/ contains two applications: authentication/ & anynote/
Also there is no api app. Every application has it's own apis inside them in a api module

egorgur committed:
add __init__.py files to apps api folders
